data/translations/es/state-data.json

```diff
@@ -117,121 +117,122 @@
     "registration_link_language_selection": "true",
     "more_info_link": "https://registertovoteflorida.gov/home",
     "more_info_link_language_selection": "true",
-    "ip_deadline": "hasta 29 días antes del día de las elecciones",
-    "online_deadline": "hasta 29 días antes del día de las elecciones",
+    "ip_deadline": "martes, 11 de octubre de 2022",
+    "online_deadline": "martes, 11 de octubre de 2022",
     "mail_received_deadline": "",
-    "mail_postmarked_deadline": "debe tener el sello del correo 29 días antes del día de las elecciones",
+    "mail_postmarked_deadline": "debe tener el sello del correo martes, 11 de octubre de 2022",
     "confirm_registration_link": "https://registration.elections.myflorida.com/es/CheckVoterStatus/Index",
     "confirm_registration_link_translated": "true"
   },
   "ga": {
     "hp_link": "https://sos.ga.gov/index.php/elections",
     "registration_link": "https://registertovote.sos.ga.gov/GAOLVR/welcome.do#no-back-button",
     "more_info_link": "https://registertovote.sos.ga.gov/GAOLVR/welcome.do#no-back-button",
-    "ip_deadline": "hasta el quinto lunes antes del día de las elecciones",
-    "online_deadline": "hasta el quinto lunes antes del día de las elecciones",
+    "ip_deadline": "martes, 11 de octubre de 2022",
+    "online_deadline": "martes, 11 de octubre de 2022",
     "mail_received_deadline": "",
-    "mail_postmarked_deadline": "debe tener el sello del correo a más tardar el quinto lunes antes del día de las elecciones",
+    "mail_postmarked_deadline": "debe tener el sello del correo martes, 11 de octubre de 2022",
     "confirm_registration_link": "https://www.mvp.sos.ga.gov/MVP/mvp.do"
   },
   "gu": {
     "hp_link": "https://gec.guam.gov/",
     "registration_link": "https://gec.guam.gov/register/",
     "more_info_link": "https://gec.guam.gov/",
-    "ip_deadline": "hasta 10 días antes del día de las elecciones",
-    "online_deadline": "hasta 10 días antes del día de las elecciones",
-    "mail_received_deadline": "a más tardar debe ser recibida 10 días antes del día de las elecciones",
+    "ip_deadline": "sábado, 15 de octubre de 2022",
+    "online_deadline": "",
```

Comment:
Does Guam no longer have an online registration deadline? If so we should remove this property because it is empty.

Reply:
Guam does not have online registration.